File: C/a015.c
#include <stdio.h>

int row = 0, col = 0 , matrix[100][100];

int main()
{
    while (scanf(" %d %d", &row, &col) == 2)
    {
        for (int i = 0; i < row; i++)
        {
            for (int j = 0; j < col; j++)
                scanf(" %d", &matrix[j][i]);
        }
        for (int i = 0; i < col; i++)
        {
            for (int j = 0; j < row; j++)
                printf("%d ", matrix[i][j]); // 反轉
            putchar('\n');
        }
    }
    return 0;
}

File: Python/a015.py
import sys

while 1:
    try:
        row, col = map(int, sys.stdin.readline().split())
    except:
        break
    table = [input().split() for r in range(row)]
    out = ''
    for c in range(col):
        for r in range(row):
            out += table[r][c] + ' '
        out = out[:-1] + '\n'
    print(out)
